bip-0174.mediawiki

```diff
@@ -198,7 +198,31 @@ The currently defined per-input types are defined as follows:
 ** Key: None. The key must only contain the 1 byte type.
 *** <tt>{0x09}</tt>
 ** Value: The UTF-8 encoded commitment message string for the proof-of-reserves.  See [[bip-0127.mediawiki|BIP 127]] for more information.
-*** <tt>{porCommitment}</tt>
+*** <tt>{preimage}</tt>
+
+* Type: RIPEMD160 preimage <tt>PSBT_RIPEMD160 = 0x0a</tt>
+** Key: The hash which the preimage must hash to
```

Comment:
> The hash which the preimage must hash to

imo, "The hash of the preimage" suffices

Reply:
Nice, I like this better. Fixed